src/tiny-invariant.ts

```diff
@@ -1,13 +1,25 @@
 const isProduction: boolean = process.env.NODE_ENV === 'production';
 const prefix: string = 'Invariant failed';
 
-// Throw an error if the condition fails
-// Strip out error messages for production
-// > Not providing an inline default argument for message as the result is smaller
+/**
+ * __invariant__
+ * 
+ * The `invariant` function takes a value, and if the value is falsy then the `invariant` function will throw. If the value is truthy, then the function will not throw.
+ *
+ * @param {boolean} condition A boolean condition - if falsey will thrown an error.
```

Comment:
the `param` can be any value

Reply:
yeah this needs to be corrected - will address